shared/middleware/properties_api.rb

```diff
@@ -24,6 +24,8 @@ class PropertiesApi < Sinatra::Base
   get %r{/v3/(shared|user)-properties/([^/]+)$} do |endpoint, channel_id|
     dont_cache
     content_type :json
+    owner_id, _ = storage_decrypt_channel_id(channel_id)
+    not_authorized unless owner_id == storage_id('user')
```

Comment:
Also if this is going to be used on other endpoints it's a good candidate for a small helper function (in super-terse Ruby style :grin:)
```ruby
not_authorized unless owns_channel? channel_id
```